app/components/Home.jsx

```diff
@@ -6,23 +6,35 @@ import bulb from './public/bulb.png';
 import controller from './public/controller.png';
 
 class Home extends PureComponent {
+    constructor(props){
+        super(props);
+        this._handleDropdownClick = this._handleDropdownClick.bind(this)
+        this.state = {
+            dropdownVisible: false
+        }
+    }
+
+    _handleDropdownClick() {
+        this.state.dropdownVisible === false ? this.setState({dropdownVisible: true}) : this.setState({dropdownVisible: false})
+    }
     render() {
         return <div className="home">
             <h1 className="_title">Yarn</h1>
-            <div className="_button-container">
+            <div className="_button-container" onClick={this._handleDropdownClick}>
                 <img className="_button -build" src={bulb} alt="build link"/>
```

Comment:
Make this a callback ie: onClick={() => this._handleDropdownClick(this.state.isDropdownVisible ? false : true)